template/main.js

```diff
@@ -1,4 +1,5 @@
 const Tram = require('tram-one')
+require("babel-polyfill")
```

Comment:
I dislike this a lot, but it is required for the polyfills to be injected

Reply:
Is this for other browser support like IE 10 and others?

Reply:
yes